### PR TITLE
For #1055: Implement Text by RqPrint

### PR DESCRIPTION
**Pull request by andreoss**
See #1055

### CODE REVIEW

**Comment by andreoss** [comment]
@paulodamaso Have a look, please

File: src/main/java/org/takes/rq/RqPrint.java [hunk 1]
```diff
@@ -29,30 +29,50 @@
...
- *
  * @since 0.1
  */
 @EqualsAndHashCode(callSuper = true)
-public final class RqPrint extends RqWrap {
+public final class RqPrint extends RqWrap implements Text {
+
+    /**
+     * The textual representation.
+     * @todo #1050:30m When (and if) cactoos/#1403 is done, change type of this
```
Comment (paulodamaso) [inline src/main/java/org/takes/rq/RqPrint.java:50]: @andreoss Seems that we won't have cactoos#1403 done, but I think we might use `Text` here anyway, WDYT?

Comment (andreoss) [reply src/main/java/org/takes/rq/RqPrint.java]: @paulodamaso Done with anonymous class. The problem here is that `TextOf` does not have a public ctor for `Scalar`, and there is no decorator which makes `Text` compute its value once.

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/takes#1056` is not assigned, can't get performer

**Pull request merged**